app/src/main/java/net/kwatts/powtools/util/BluetoothUtilImpl.java

```diff
@@ -265,19 +270,20 @@ public void onScanResult(int callbackType, ScanResult result) {
             String deviceName = result.getDevice().getName();
             String deviceAddress = result.getDevice().getAddress();
 
-            Log.i(TAG, "ScanCallback.onScanResult: " + mScanResults.entrySet());
+            Timber.i( "ScanCallback.onScanResult: " + mScanResults.entrySet());
             if (!mScanResults.containsKey(deviceAddress)) {
-                Log.i(TAG, "ScanCallback.deviceName:" + deviceName);
+                Timber.i( "ScanCallback.deviceName:" + deviceName);
                 mScanResults.put(deviceAddress, deviceName);
 
                 if (deviceName == null) {
-                    updateLog("Found " + deviceAddress);
+                    Timber.i("Found " + deviceAddress);
                 } else {
-                    updateLog("Found " + deviceAddress + " (" + deviceName + ")");
+                    Timber.i("Found " + deviceAddress + " (" + deviceName + ")");
                 }
 
                 if (deviceName != null && (deviceName.startsWith("ow") || deviceName.startsWith("Onewheel"))) {
-                    updateLog("Looks like we found our OW device (" + deviceName + ") discovering services!");
```

Comment:
taking out the user viewable log? or was it too verbose? did you want to take it all the way out eventually?

Reply:
Want to take it out, IMO the UI should stand on its own, shouldn't need it. For now if debug is checked we can show the debugdrawer... that will allow us to be more smart with logging :)